### PR TITLE
Donald Dec Docs Update

### PR DESCRIPTION
**Pull request by DonaldChung-HK**
Hi Alex,

I just moved all my work from confluence to Readthedocs. Below is a summary of changes that I made to it.

Added docs
- Rebuild VM
- Resize VM
- Delete VM
- SnapshotVM
- Creating Snapshot From Image
- Volume (Create, Delete, Attach (and access), Detach, Snapshot)
- Security Group (Create, Delete and rule customisation)
Removed redundant doc files
- Create Volume with CLI and Web Interface (grouped into volume page)
- Create image from snapshot (grouped into SnapshotVM page)

Thanks,

Donald

### CODE REVIEW

File: source/howto/SnapshotVM.rst [hunk 1]
```diff
@@ -1,5 +1,183 @@
-=================================
-Snapshot a VM to create an image
-=================================
-
-Documentations coming soon
+**********************
+Instance Snapshot
```
Comment (apdibbo) [inline source/howto/SnapshotVM.rst]: This title isnt obvious what the document is for, the previous title was more informative

File: source/howto/Volume.rst [hunk 2]
```diff
@@ -0,0 +1,426 @@
+Volumes in OpenStack
```
Comment (apdibbo) [inline source/howto/Volume.rst:2]: This looks like it should be a title (a = line before and after)

**Review by apdibbo — CHANGES_REQUESTED** [review]
A couple of minor comments but looks good in general

File: source/howto/SnapshotVM.rst (hunk 1 above)
Comment (DonaldChung-HK) [reply source/howto/SnapshotVM.rst]: Replaced title

File: source/howto/Volume.rst (hunk 2 above)
Comment (DonaldChung-HK) [reply source/howto/Volume.rst:2]: Added the title line

**Review by apdibbo — APPROVED** [review]

**Pull request merged**